test/e2e/install/install_test.go

```diff
@@ -150,13 +146,41 @@ func mayastorReadyPodCount() int {
 	return int(mayastorDaemonSet.Status.NumberAvailable)
 }
 
-func moacReadyPodCount() int {
+func moacReady() bool {
 	var moacDeployment appsV1.Deployment
 	if k8sClient.Get(context.TODO(), types.NamespacedName{Name: "moac", Namespace: "mayastor"}, &moacDeployment) != nil {
-		fmt.Println("Failed to get MOAC deployment")
-		return -1
+		logf.Log.Info("Failed to get MOAC deployment")
+		return false
 	}
-	return int(moacDeployment.Status.AvailableReplicas)
+
+	// { Remove/Reduce verbosity once we have fixed install test occasional failure.
```

Comment:
Off topic for this PR, but I would say that the e2e test suite is rather very scarce of relevant logs. I would definitely suggest not lowering verbosity. Solution might be to use more logging levels to leave that for a user (this might be Log.Debug, and for loop below might log Log.Trace) and allow setting verbosity on command line.